pkg/varlinkapi/transfers.go

```diff
@@ -39,7 +40,7 @@ func (i *VarlinkAPI) SendFile(call iopodman.VarlinkCall, ftype string, length in
 
 	logrus.Debugf("successfully received %s", outputFile.Name())
 	// Send an ACK to the client
-	call.Call.Writer.WriteString(outputFile.Name())
+	call.Call.Writer.WriteString(fmt.Sprintf("%s:", outputFile.Name()))
```

Comment:
```suggestion
	call.Call.Writer.WriteString(outputFile.Name() +":")
```
Uses less resources and is faster.

Reply:
ill do this in master, this is really just a line revert.